### ISSUE TITLE
config load_minigraph fails due to Redis BGSAVE already in progress

### ISSUE THREAD

**Issue by lolyu** [Azure/sonic-buildimage#6498]
**Description**
config load_minigraph fails due to Redis BGSAVE already in progress

**Steps to reproduce the issue:**
1. run `config load_minigraph`

**Describe the results you received:**
```
admin@str-a7060cx-acs-1:~$ sudo config load_minigraph -y
Executing stop of service telemetry...
Warning: Stopping telemetry.service, but it can still be activated by:
  telemetry.timer
Executing stop of service swss...
Executing stop of service lldp...
Executing stop of service pmon...
Executing stop of service bgp...
Executing stop of service hostcfgd...
Running command: /usr/local/bin/sonic-cfggen -H -m -j /etc/sonic/init_cfg.json   --write-to-db
Running command: pfcwd start_default
Running command: config qos reload
Traceback (most recent call last):
  File "/usr/local/bin/sonic-cfggen", line 416, in <module>
    main()
  File "/usr/local/bin/sonic-cfggen", line 405, in main
    configdb.mod_config(FormatConverter.output_to_db(data))
  File "/usr/local/lib/python2.7/dist-packages/swsssdk/configdb.py", line 416, in mod_config
    client.bgsave()
  File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 695, in bgsave
    return self.execute_command('BGSAVE')
  File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 668, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/usr/local/lib/python2.7/dist-packages/redis/client.py", line 680, in parse_response
    response = connection.read_response()
  File "/usr/local/lib/python2.7/dist-packages/redis/connection.py", line 629, in read_response
    raise response
redis.exceptions.ResponseError: Background save already in progress
```

**Describe the results you expected:**
No such an error.

**Additional information you deem important (e.g. issue happens only occasionally):**

```
admin@str-a7060cx-acs-1:~$ show version

SONiC Software Version: SONiC.20191130.58
Distribution: Debian 9.13
Kernel: 4.9.0-11-2-amd64
Build commit: 089214873
Build date: Thu Jan  7 10:53:18 UTC 2021
Built by: sonicbld@jenkins-slave-phx-2

Platform: x86_64-arista_7060_cx32s
HwSKU: Arista-7060CX-32S-D48C8
ASIC: broadcom
Serial Number: JPE17051884
Uptime: 03:40:59 up  1:11,  2 users,  load average: 0.22, 1.03, 1.61

Docker images:
REPOSITORY                 TAG                 IMAGE ID            SIZE
docker-snmp-sv2            20191130.58         46de4ef67f1f        348MB
docker-snmp-sv2            latest              46de4ef67f1f        348MB
docker-fpm-frr             20191130.58         21296f4b1f64        326MB
docker-fpm-frr             latest              21296f4b1f64        326MB
docker-acms                20191130.58         89991569477f        194MB
docker-acms                latest              89991569477f        194MB
docker-lldp-sv2            20191130.58         04a96b151640        312MB
docker-lldp-sv2            latest              04a96b151640        312MB
docker-orchagent           20191130.58         810135ffcf83        333MB
docker-orchagent           latest              810135ffcf83        333MB
docker-syncd-brcm          20191130.58         2a961a57ee57        437MB
docker-syncd-brcm          latest              2a961a57ee57        437MB
docker-teamd               20191130.58         748227bc0de0        315MB
docker-teamd               latest              748227bc0de0        315MB
docker-platform-monitor    20191130.58         548e262b3f7e        358MB
docker-platform-monitor    latest              548e262b3f7e        358MB
docker-sonic-telemetry     20191130.58         4deeec7b321a        353MB
docker-sonic-telemetry     latest              4deeec7b321a        353MB
docker-database            20191130.58         c19667071be6        290MB
docker-database            latest              c19667071be6        290MB
docker-router-advertiser   20191130.58         642b7f38f1b4        290MB
docker-router-advertiser   latest              642b7f38f1b4        290MB
docker-dhcp-relay          20191130.58         442975d2e863        300MB
docker-dhcp-relay          latest              442975d2e863        300MB
k8s.gcr.io/pause           3.2                 80d28bedfe5d        683kB
```

**Comment by lolyu** [comment]
This looks like the `bgsave`  [here](https://github.com/Azure/sonic-py-swsssdk/blob/591a5c0bda1ce4c51fe03f60646014d03172040b/src/swsssdk/configdb.py#L416) is the reason for this. This cannot guarantee there is no `save` running in background so `Redis` warns about this.

**Comment by anshuv-mfst** [comment]
@tahmed-dev could you please look into this issue, thanks.

**Comment by tahmed-dev** [comment]
bgsave was removed via [PR:86](https://github.com/Azure/sonic-py-swsssdk/pull/86) I added a label to request the fix to be ported to 201911.

**Comment by tahmed-dev** [comment]
@abdosi FYI...

**Comment by tahmed-dev** [comment]
@lolyu can you please close this issue once the fix is verified?

**Comment by lolyu** [comment]
@tahmed-dev, no problem.

**Comment by lolyu** [comment]
Close it since the issue is resolved in `20191130.59` and `config load_minigraph` works well.
```
admin@str-a7060cx-acs-10:~$ sudo config load_minigraph -y
Executing stop of service telemetry...
Warning: Stopping telemetry.service, but it can still be activated by:
  telemetry.timer
Executing stop of service swss...
Executing stop of service lldp...
Executing stop of service pmon...
Executing stop of service bgp...
Executing stop of service hostcfgd...
Running command: /usr/local/bin/sonic-cfggen -H -m -j /etc/sonic/init_cfg.json   --write-to-db
Running command: pfcwd start_default
Running command: config qos reload
Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-arista_7060_cx32s/Arista-7060CX-32S-C32/buffers.json.j2 > /tmp/buffers.json
Running command: /usr/local/bin/sonic-cfggen  -d -t /usr/share/sonic/device/x86_64-arista_7060_cx32s/Arista-7060CX-32S-C32/qos.json.j2 -y /etc/sonic/sonic_version.yml > /tmp/qos.json
Running command: /usr/local/bin/sonic-cfggen  -j /tmp/buffers.json --write-to-db
Running command: /usr/local/bin/sonic-cfggen  -j /tmp/qos.json --write-to-db

Executing reset-failed of service bgp...
Executing reset-failed of service dhcp_relay...
Executing reset-failed of service hostcfgd...
Executing reset-failed of service hostname-config...
Executing reset-failed of service interfaces-config...
Executing reset-failed of service lldp...
Executing reset-failed of service ntp-config...
Executing reset-failed of service pmon...
Executing reset-failed of service radv...
Executing reset-failed of service rsyslog-config...
Executing reset-failed of service snmp...
Executing reset-failed of service swss...
Executing reset-failed of service syncd...
Executing reset-failed of service teamd...
Executing reset-failed of service telemetry...
Executing restart of service hostname-config...
Executing restart of service interfaces-config...
Executing restart of service ntp-config...
Executing restart of service rsyslog-config...
Executing restart of service swss...
Executing restart of service bgp...
Executing restart of service pmon...
Executing restart of service lldp...
Executing restart of service hostcfgd...
Executing restart of service telemetry...
Reloading Monit configuration ...
Reinitializing monit daemon

Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`.
```
